Tests/MapboxCoreNavigationTests/MapboxCoreNavigationTests.swift

```diff
@@ -23,6 +23,17 @@ let waitForInterval: TimeInterval = 5
 class MapboxCoreNavigationTests: XCTestCase {
     var navigation: MapboxNavigationService!
     
+    override func setUp() {
+        super.setUp()
+        UserDefaults.standard.set("Location Usage Description", forKey: "NSLocationWhenInUseUsageDescription")
+        UserDefaults.standard.set("Location Usage Description", forKey: "NSLocationAlwaysAndWhenInUseUsageDescription")
```

Comment:
Sorry for being unclear: my suggestion in https://github.com/mapbox/mapbox-navigation-ios/pull/2876#discussion_r603571053 was to simply set a generic flag like `MBXIsTestingCoreNavigation` to `true` and checking that it’s true in `checkForLocationUsageDescription()`. However, this approach is even better in a way; we can follow this pattern in the future if there’s any code that cares about the actual value of the key, not just whether it’s present. 👍